Resources/Prototypes/Entities/Objects/Specific/Medical/hypospray.yml

```diff
@@ -657,18 +657,23 @@
   components:
   - type: SolutionContainerManager
     solutions:
-      hypospray:
+      injector: # Starlight edit
```

Comment:
If you change the SolutionName on the added InjectorComponent on this prototype to "hypospray", you'd be able to avoid these first three lines of potential conflicts.